vscode/src/statusbar.ts

```diff
@@ -89,27 +90,40 @@ function registerTargetProfileCommand() {
 }
 
 const targetProfiles = [
-  { configName: "base", uiText: "QIR:Base" },
-  { configName: "unrestricted", uiText: "Unrestricted" },
+  { configName: "base", uiText: "Q#: QIR base" },
+  { configName: "unrestricted", uiText: "Q#: unrestricted" },
 ];
 
 function getTargetProfileUiText(targetProfile?: string) {
   switch (targetProfile) {
     case "base":
-      return "QIR:Base";
+      return "Q#: QIR base";
```

Comment:
Out of interest, why do we repeat the string literal here, rather than use the targetProfiles const map that includes it in lines 93/94 above?

Reply:
(I guess that would be a little awkward its the value not the key to look up on, so never mind)